Mathlib/Geometry/Manifold/VectorBundle/Tangent.lean

```diff
@@ -124,6 +124,47 @@ theorem tangentBundleCore_coordChange_achart (x x' z : M) :
   rfl
 #align tangent_bundle_core_coord_change_achart tangentBundleCore_coordChange_achart
 
+section tangentCoordChange
+
+/-- Since `tangentBundleCore` has the same base sets as the preferred charts of the base manifold,
+  we define `tangentCoordChange I x y` as a convenient abbreviation for a change of coordinates
+  between the preferred charts at `x y : M`. -/
```

Comment:
I think I'd reverse the order of emphasis here and start with the fact that `x` and `y` are really indexing charts. Maybe something like:
```suggestion
/-- In a manifold `M`, given two preferred charts indexed by `x y : M`, `tangentCoordChange I x y` is the family of derivatives of the corresponding change-of-coordinates map. It takes junk values outside the intersection of the sources of the two charts.

Note that this definition takes advantage of the fact that `tangentBundleCore` has the same base sets as the preferred charts of the base manifold. -/
```